hack/jenkins/gotest-dockerized.sh

```diff
@@ -36,6 +36,9 @@ export KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/_artifacts
 # From _inside_ a container that has the host's docker mapped in, the $PATH
 # provided must be resolvable on the *HOST*, not the container.
 
+# TODO: either push docker people to fix --rm=true option or clean up images after tests
```

Comment:
I think that you should simply add those two lines at the beginning:

    docker rm -v $(docker ps -a -q -f status=exited)
    docker rmi $(docker images -f "dangling=true" -q)

Than you will not need any cron job, etc.